pyproject.toml

```diff
@@ -19,6 +19,7 @@ classifiers=[
     'Programming Language :: Python :: 3.8',
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
```

Comment:
**Non-blocking** Do you think we could remove the python 3.7 above. I don't think thats been tested in the actions.

Reply:
And below wagtail 2 and 3 could be removed

Reply:
Since I included the Python 3.7 test in the tox.ini and used tox-gh-actions, I retained the Python 3.7 classifier. I removed Wagtail 2 and 3.

I also deleted the setup.py since pyproject.toml is present.

Reply:
Great thank you.